src/constants/env.ts

```diff
@@ -0,0 +1,6 @@
+export const env = {
+  KAKAO_REST_API_KEY: process.env.NEXT_PUBLIC_KAKAO_REST_API_KEY as string,
+  KAKAO_REDIRECT_URI: process.env.NEXT_PUBLIC_KAKAO_REDIRECT_URI as string,
+  AUTH_TOKEN_KEY: process.env.NEXT_PUBLIC_AUTH_TOKEN_KEY as string,
+  API_MOCKING: process.env.NEXT_PUBLIC_API_MOCKING as string
```

Comment:
```suggestion
  KAKAO_REST_API_KEY: process.env.NEXT_PUBLIC_KAKAO_REST_API_KEY  || '',
  KAKAO_REDIRECT_URI: process.env.NEXT_PUBLIC_KAKAO_REDIRECT_URI  || '',
  AUTH_TOKEN_KEY: process.env.NEXT_PUBLIC_AUTH_TOKEN_KEY  || '',
  API_MOCKING: process.env.NEXT_PUBLIC_API_MOCKING  || ''
```
env를 기반으로 하는 코드를 강제 형변환 하는 경우, 런타임에서 에러가 발생할 수 있을 것 같아서 차라리 기본값을 넣는 게 좋을 것 같아요! (ex. .env 파일이 없는데 string으로 추론)

Reply:
[변경 완료](https://github.com/price-offer/offer-fe/pull/154/commits/0efc8a2edb02bacd7a277b6267096a456a917bb3) !